tests/foreman/ui/test_rhcloud_insights.py

```diff
@@ -300,6 +302,8 @@ def test_host_details_page(
 
 
 @pytest.mark.e2e
+@pytest.mark.pit_server
```

Comment:
Marking this test with `pit_server` might be an issue since our internal infra does not have an `install-capsule-on-top-of-rhel-compose` workflow defined. We need to define that workflow first and also make sure that our PIT pipeline script changes the capsule config as desired. 

Without this workflow, the capsule would be installed on top of the GA'd RHEL version instead of the RHEL version that we are supposed to test on.

Hit me up on Slack, I can guide you.